JavaScript/JavaScriptSDK/Util.ts

```diff
@@ -453,7 +453,7 @@ module Microsoft.ApplicationInsights {
 
         public static parseUrl(url): HTMLAnchorElement {
             if (!UrlHelper.htmlAnchorElement) {
-                UrlHelper.htmlAnchorElement = UrlHelper.document.createElement('a');
+                UrlHelper.htmlAnchorElement = !!UrlHelper.document.createElement ? UrlHelper.document.createElement('a'): {};
```

Comment:
So you're saying if it's non-browser then noop is fine because we don't guarantee it working there/sdk user is responsible for making sure it works.